### DISCUSSION TITLE
sort/filter by ranked in song select

### DISCUSSION THREAD

**Discussion by Axilotl17** [ppy/osu#18593] [Ideas] [answered]
sometimes you only want to play ranked songs, or graveyarded or loved songs. or, looking for a specific map in that category. either way, filtering by map type would be useful in song select. 

you could have a few default locked collections, like one for each state a map can be in, and these can't be edited.

**Comment by Theighlin** [comment, marked as answer]
You can already do this by searching "status=x" in song select, where x can be r for ranked, l for loved. (I assume it works for graveyarded maps too).

Were you aware of this, or do you think that is not enough?

**Reply by Axilotl17** [reply]
i was unaware, but i do still think some pre-made collections would make it easier... is there a list of these search filters on the wiki?
